src/components/Article.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => (
+  <>
```

Comment:
```suggestion
  <article>
```
Why would you use fragment tag if there is more semantical tag available?